src/components/PackTile.js

```diff
@@ -73,95 +73,122 @@ const PackTile = ({
       className={className}
       interactive
       css={css`
+        --tile-image-height: 100px;
+        padding: 1rem;
         overflow: hidden;
-        display: flex;
-        flex-direction: column;
-
-        ${view === VIEWS.LIST &&
+        display: grid;
+        grid-gap: 0.2rem;
+        grid-template-rows: var(--tile-image-height) 3rem 1fr 0.2fr;
```

Comment:
Perhaps we want to add a comment to what each of these row heights represent (image, title, desc, chip).